[2025-09-25T20:26:35.728+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T20:26:35.735+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hw1.collect_metrics_model manual__2025-09-25T20:21:14.086804+00:00 [queued]>
[2025-09-25T20:26:35.738+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hw1.collect_metrics_model manual__2025-09-25T20:21:14.086804+00:00 [queued]>
[2025-09-25T20:26:35.738+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-09-25T20:26:35.743+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): collect_metrics_model> on 2025-09-25 20:21:14.086804+00:00
[2025-09-25T20:26:35.747+0000] {standard_task_runner.py:64} INFO - Started process 71958 to run task
[2025-09-25T20:26:35.752+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hw1', 'collect_metrics_model', 'manual__2025-09-25T20:21:14.086804+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/kudryakov_hw1-checkpoint.py', '--cfg-path', '/tmp/tmpaxf_2rom']
[2025-09-25T20:26:35.759+0000] {standard_task_runner.py:91} INFO - Job 176: Subtask collect_metrics_model
[2025-09-25T20:26:35.786+0000] {task_command.py:426} INFO - Running <TaskInstance: hw1.collect_metrics_model manual__2025-09-25T20:21:14.086804+00:00 [running]> on host 2b301b74205f
[2025-09-25T20:26:35.823+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='VLADISLAV KUDRYAKOV' AIRFLOW_CTX_DAG_ID='hw1' AIRFLOW_CTX_TASK_ID='collect_metrics_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T20:21:14.086804+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-25T20:21:14.086804+00:00'
[2025-09-25T20:26:35.824+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T20:26:35.830+0000] {logging_mixin.py:188} INFO - collect_metrics_model
[2025-09-25T20:26:35.830+0000] {kudryakov_hw1-checkpoint.py:203} INFO - Начали расчет метрик
[2025-09-25T20:26:35.838+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2025-09-25T20:26:35.838+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2025-09-25T20:26:39.315+0000] {logging_mixin.py:188} INFO - {'r_squared': 0.8503834409713716, 'RMSE': 0.19244684232083303} metrics
[2025-09-25T20:26:39.579+0000] {kudryakov_hw1-checkpoint.py:222} INFO - Успешно Завершен расчет метрик
[2025-09-25T20:26:39.602+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-25T20:26:39.602+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T20:26:39.610+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hw1, task_id=collect_metrics_model, run_id=manual__2025-09-25T20:21:14.086804+00:00, execution_date=20250925T202114, start_date=20250925T202635, end_date=20250925T202639
[2025-09-25T20:26:39.636+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T20:26:39.649+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-25T20:26:39.650+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
